stablehlo/dialect/TypeInference.cpp

```diff
@@ -242,13 +244,15 @@ LogicalResult verifyBatchNorm(std::optional<Location> location,
   if (featureIndex < 0)
     return emitOptionalError(location, "expects featureIndex to be a ",
                              "non-negative number, got ", featureIndex, ".");
-  // Note: the above checks '0 <= feature-index < multiDimType.getRank()'
+  // Note: the above checks '0 <= feature_index < multiDimType.getRank()'
```

Comment:
Is this already existing comment relevant?

Reply:
Not really, will remove.